api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -57,44 +96,63 @@
 """
 Provider = Literal["S3", "file", "unknown"]
 
-"""
-A mirror identifies a location that can host the census artifacts. A dict of available mirrors exists
-in the mirrors.json file, and looks like this:
-
-{
-    "default": "default-mirror",
-    "default-mirror": {
-        "provider": "S3",
-        "base_uri": "s3://a-public-bucket/",
-        "region": "us-west-2"
-    }
-}
 
-"""
 CensusMirrorName = str  # name of the mirror
-CensusMirror = TypedDict(
-    "CensusMirror",
-    {
-        "provider": Provider,  # provider of the mirror.
-        "base_uri": str,  # base URI for the mirror location, e.g. s3://cellxgene-data-public/
-        "region": Optional[str],  # region of the bucket or resource
-    },
-)
+
+
+class CensusMirror(TypedDict):
+    """A mirror for a Census resource.
+
+    A mirror identifies a location that can host the census artifacts. A dict of available mirrors exists in the
+    ``mirrors.json`` file, and looks like this:
+
+    .. highlight:: json
+    .. code-block:: json
+
+        {
+            "default": "default-mirror",
+            "default-mirror": {
+                "provider": "S3",
+                "base_uri": "s3://a-public-bucket/",
+                "region": "us-west-2"
+            }
+        }
+
+    Args:
+        provider:
+            provider of the mirror.
```

Comment:
I am likely the original person to blame, but we should be consistent with the trailing dots, especially now that these are docstrings and not comments. Also, I'd capitalize these. The [Google-style docstrings guidelines](https://google.github.io/styleguide/pyguide.html) seem to suggest both capitalization and trailing dots, even for small sentences, so I'd stick with those.

Note that we may want to a pass to every file as I noticed there are inconsistencies elsewhere. I'm happy to do that in a separate PR.

Reply:
Resolved, at least for this file

Reply:
Thanks! I'll take care of the other files.